source/math/FloatPoint2DVectorList.ooc

```diff
@@ -87,6 +87,54 @@ FloatPoint2DVectorList: class extends VectorList<FloatPoint2D> {
 		}
 		result
 	}
+	offset: func (offset: FloatPoint2D) -> This {
```

Comment:
Maybe actually addOffset is better as offset might sound like you are shifting it... I don't know... let's maybe just keep it like this?